src/Stratis.Bitcoin.Features.MemoryPool.Tests/MempoolValidatorTest.cs

```diff
@@ -496,10 +826,40 @@ public void AcceptToMemoryPool_TxInputsAreBad_ReturnsFalse()
             // TODO: Execute this case CheckMempoolCoinView !context.View.HaveInputs(context.Transaction)
         }
 
-        [Fact(Skip = "Not implemented yet.")]
-        public void AcceptToMemoryPool_NonBIP68CanMine_ReturnsFalse()
+        [Fact]
+        public async void AcceptToMemoryPool_NonBIP68CanMine_ReturnsFalseAsync()
         {
-            // TODO: Execute this case CreateMempoolEntry !CheckSequenceLocks(this.chain.Tip, context, PowCoinViewRule.StandardLocktimeVerifyFlags, context.LockPoints)
+            string dataDir = GetTestDirectoryPath(this);
+
+            // Run mempool tests on mainnet so that RequireStandard flag is set in the mempool settings.
+            Network network = KnownNetworks.Main;
+            var minerSecret = new BitcoinSecret(new Key(), network);
+            ITestChainContext context = await TestChainFactory.CreateAsync(network, minerSecret.PubKey.Hash.ScriptPubKey, dataDir);
+            IMempoolValidator validator = context.MempoolValidator;
+            Assert.NotNull(validator);
+
+            var destSecret = new BitcoinSecret(new Key(), network);
+            var tx = new Transaction();
+
+            // Need version >= 2 for BIP68 to be enforced.
+            tx.Version = 2;
+
+            tx.AddInput(new TxIn(new OutPoint(context.SrcTxs[0].GetHash(), 0), PayToPubkeyHashTemplate.Instance.GenerateScriptPubKey(minerSecret.PubKey)));
+            tx.AddOutput(new TxOut(new Money(Money.Coins(1)), destSecret.PubKeyHash));
+
+            // Per BIP68 the nSequence value of an input can be used to restrict its spendability
+            // prior to a certain timestamp or block height. Here we set it to an arbitrary high
+            // value to trigger a BIP68 check failure.
+            tx.Inputs.First().Sequence = new Sequence(5000);
```

Comment:
do we have a test also on time not just height?

Reply:
I will add a stub for it so it doesn't get forgotten, it may be quick to copy the bulk of this test and tweak it for time.